cpp/src/gandiva/precompiled/arithmetic_ops_test.cc

```diff
@@ -101,6 +101,34 @@ TEST(TestArithmeticOps, TestDiv) {
   context.Reset();
 }
 
+TEST(TestArithmeticOps, TestGreatestLeast) {
+  // Comparable functions - Greatest and Least
+  EXPECT_EQ(greatest_int32_int32(1, 2), 2);
```

Comment:
can you add some unit tests for other types too?

Reply:
I added the types int64, float32 and float64 to the unit tests